src/sbt-test/sbt-plugins/simple/test

```diff
@@ -0,0 +1,29 @@
+# See http://eed3si9n.com/testing-sbt-plugins for sbt-scripted test syntax
+# initialize a git repository to make our plugins happy
+$ exec git init
+$ exec git add .
+$ exec git commit -m "initial commit"
+
+> checkFormat
+
+# make sure all our code compiles, including:
+# core/src/it test code that depends on core/src/test code
+> it:compile
+
+# Style plugin tests
+> checkStyle
+> styleCheckStrict
+-> test:styleCheckStrict
+
+# webapp plugin tests
+-$ exists client/build/index.html
+-$ exists client/node_modules
+> webApp/npm:build
+$ exists client/build/index.html
+$ exists client/node_modules
+
+# webservice plugin tests
```

Comment:
It would be great if you replicated one or more of the cache key tests here as well.

Verifying that the key changes when the source code changes, verifying that it changes if a dependency changes, and verifying that it changes if a local dependency changes are the three tests. I think. :)